### PR TITLE
Use React.lazy for loading components

### PR DESCRIPTION
**Pull request by antoinechalifour**
**Type:** Refactor

The following has been addressed in the PR: Use `React.Lazy` for loading components

*  There is a related issue? Nop
*  Unit or Functional tests are included in the PR None changed

### CODE REVIEW

**Comment by juanpicado** [comment]
Could you share any analysis that prove the impact of this changes in the bundle size? chunks etc etc ?

**Comment by antoinechalifour** [comment]
I'll try to give you some numbers tomorrow but here's the short version :

- using `React.lazy` is the idiomatic way of doing code splitting in react land
- it removed the `asyncComponent` file so the bundle should be lighter

**Comment by priscilawebdev** [comment]
@antoinechalifour Well done! 👏 Maybe you could also remove the file "asynComponent", because it is not being used anymore

File: src/router.tsx [hunk 1]
```diff
@@ -25,7 +24,7 @@ class RouterApp extends Component<RouterAppProps> {
   public render(): ReactElement<HTMLDivElement> {
     return (
       <Router history={history}>
-        <>
+        <React.Suspense fallback={null}>
```
Comment (priscilawebdev) [inline src/router.tsx]: Please add the loading component here
```suggestion
        <React.Suspense fallback={<Loading />}>
```

Comment (antoinechalifour) [reply src/router.tsx]: For now the `asyncComponent` renders `null` so I wanted to keep the existing behavior. But I can change it to display a loading component

**Comment by antoinechalifour** [comment]
I think it's already been removed ! I'll check tomorrow

File: src/router.tsx (hunk 1 above)
Comment (juanpicado) [reply src/router.tsx]: `<Loading/>` already are defined in each page, we can skip this.

**Pull request merged**